src/databricks/azext_databricks/_params.py

```diff
@@ -0,0 +1,55 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_three_state_flag,
+    get_enum_type,
+    get_location_type
+)
+
+# from azext_databricks.action import (
+#     PeeringAddAuthorizations
+# )
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('databricks workspace create') as c:
+        c.argument('workspace_name', options_list=['--name', '-n'], help='The name of the workspace.')
+        c.argument('tags', tags_type)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx))
+        c.argument('managed_resource_group', help='The managed resource group to create. It can be either a name or a resource id.')
+        c.argument('aml_workspace_id', help='Azure Machine Learning workspace id.')
+        c.argument('custom_virtual_network_id', options_list=['--virtual-network-id'], help='Virtual network id.')

```

Comment:
just curious, why vnet-id, but subnet-name? Suppose both should accept both name and resource id, right? and suggested way is just resource instead of resource-name

Reply:
subnet-name is the name for a new subnet to be created, so id is not necessary. I added name or id for virtual-network.